website/static/js/home-page/newAndNoteworthyPlugin.js

```diff
@@ -43,12 +43,17 @@ var NewAndNoteworthy = {
         var newAndNoteworthyUrl = $osf.apiV2Url('nodes/' + window.contextVars.newAndNoteworthy + '/node_links/', {});
         var newAndNoteworthyPromise = m.request({method: 'GET', url: newAndNoteworthyUrl, config: xhrconfig, background: true});
         newAndNoteworthyPromise.then(function(result){
-            var numNew = Math.min(result.data.length, self.SHOW_TOTAL);
+            var numNew = result.data.length;
             for (var l = 0; l < numNew; l++) {
                 var data = result.data[l];
                 if (lodashGet(data, 'embeds.target_node.data', null)) {
-                    self.newAndNoteworthyNodes().push(result.data[l]);
-                    self.fetchContributors(result.data[l]);
+                    if (data.embeds.target_node.data.attributes.public === true) {
```

Comment:
Are you talking about changing how node links work?  The projects under Discover Public Projects are node links.  Current behavior, it someone makes a node link private, the project stays linked, you just no longer have access to it.  

We are fetching the node links on the node via API v2 and some could be private.